packages/react-components/theme-designer/src/components/Sidebar/Sidebar.tsx

```diff
@@ -188,13 +227,14 @@ export const Sidebar: React.FC<SidebarProps> = props => {
         min={0}
         max={100}
         id={lightCpId}
-        value={props.lightCp * 100}
+        value={custom.lightCp * 100}
         onChange={handleLightCpChange}
       />
       <Label htmlFor={darkCpId}>Dark Control Point</Label>
-      <Slider size="small" min={0} max={100} id={darkCpId} value={props.darkCp * 100} onChange={handleDarkCpChange} />
+      <Slider size="small" min={0} max={100} id={darkCpId} value={custom.darkCp * 100} onChange={handleDarkCpChange} />
+      <Switch onChange={toggleTheme} label={custom.isDark ? 'dark theme' : 'light theme'} />
```

Comment:
The switch is not working, when you switch between dark and light it stays "off".

Reply:
really sorry, i forgot to mention but yes, memoizing the switch + rest of the features in the sidebar is a goal for the next PR :)

Reply:
Do we have a follow-up issue filed for this that we could link to the PR description?

Reply:
skipping because there are plans for a quick follow up